Sources/Plasma/PubUtilLib/plPipeline/GL/plGLMaterialShaderRef.cpp

```diff
@@ -647,7 +647,7 @@ void plGLMaterialShaderRef::ILoopOverLayers()
     for (j = 0; j < fMaterial->GetNumLayers(); )
     {
         size_t iCurrMat = j;
-        std::shared_ptr<plShaderFunction> fragPass = std::make_shared<plShaderFunction>(ST::format("pass{}", pass), "void");
+        std::shared_ptr<plShaderFunction> fragPass = std::make_shared<plShaderFunction>(ST::format("pass{}", pass), "vec4");
         std::shared_ptr<plShaderFunction> vertPass = std::make_shared<plShaderFunction>(ST::format("pass{}", pass), "void");
 
         j = IHandleMaterial(iCurrMat, vertPass, fragPass);
```

Comment:
Changing the pass0 function so that it returns the color as a vec4 instead of trying to set it directly. There were some warnings online about trying to set the fragment color return value outside of main.